src/test/groovy/nebula/test/SpecifiedGradleVersionIntegrationSpec.groovy

```diff
@@ -23,7 +24,12 @@ class SpecifiedGradleVersionIntegrationSpec extends IntegrationSpec {
         when:
             def result = runTasksSuccessfully('build')
         then:
-            result.standardOutput.contains("gradle/$requestedGradleVersion/taskArtifacts")
+            if (Os.isFamily(Os.FAMILY_WINDOWS)){
```

Comment:
There is a spock construct to do this. I'd rather have 2 separate tests that require different OS's then an if block in the test.

http://spockframework.org/spock/docs/1.0/extensions.html

IgnoreIf block

```
os Information about the operating system (see spock.util.environment.OperatingSystem)
```